conda/conda-reqs-pip.txt

```diff
@@ -1,7 +1,8 @@
-azure-mgmt-monitor>=2.0.0
 azure-mgmt-resourcegraph>=8.0.0
-azure-mgmt-subscription>=3.0.0
-KqlmagicCustom[jupyter-basic,auth_code_clipboard]>=0.1.114.post22
+azure-monitor-query>=1.0.0, <=2.0.0
+# KqlmagicCustom[jupyter-basic,auth_code_clipboard]>=0.1.114.post22
```

Comment:
Whats the reason for commenting out this?

Reply:
I'm hoping to remove dependency on Kqlmagic. It's still available as an option but unlike the other optional items in here it has a lot of dependencies. People will get a warning if they try to use one of the deprecated components that still use it and should have enough info to get it installed.